tools/rndf_visualizer/src/MapLanes.cc

```diff
@@ -1151,38 +1162,67 @@ void createEquilateralTriangle(svg::Polygon &triangle, float x, float y, float r
 }
 
 
-void drawWaypoint(svg::Document &doc, float ratio, float min_x, float max_y, float angle, WayPointNode &w1){
+void drawWaypoint(svg::Document &doc, float ratio, float min_x, float max_y, float angle, WayPointNode &w1, bool flat, svg::Fill *defaultColor){
 	static svg::Fill greenFill = svg::Fill(svg::Color::Green);
```

Comment:
We can reuse it but indexes will be hardcoded and color array needs to be global. I've preferred to keep them separate also because one is for creating lane stroke with colors and the other are color fills for polygons.